shared/hedvig-ui/popover.tsx

```diff
@@ -0,0 +1,76 @@
+import React, { useState } from 'react'
+import styled, { css } from 'react-emotion'
+
+const PopoverWrapper = styled.div`
+  position: relative;
+  display: inline-block;
+`
+
+const Contents = styled.div`
+  position: absolute;
+  top: 0;
+  left: 50%;
+  transform: translate(-50%, calc(-100% - 0.25rem));
+  padding: 0.5rem;
+  border-radius: 0.25rem;
+
+  ${({ theme }) => css`
+    background: ${theme.foreground};
+    color: ${theme.background};
+  `}
+`
```

Comment:
Plain css does the trick :) 

```suggestion
const Contents = styled.div`
    display: none;
  position: absolute;
  top: 0;
  left: 50%;
  transform: translate(-50%, calc(-100% - 0.25rem));
  padding: 0.5rem;
  border-radius: 0.25rem;

  ${PopoverWrapper}:hover & {
    display: block;
  }

  ${({ theme }) => css`
    background: ${theme.foreground};
    color: ${theme.background};
  `}
`
`
```